### ISSUE TITLE
Google Drive not working

### ISSUE THREAD

**Issue by mateusz-szafraniec** [iPaulPro/aFileChooser#40]
Hi, as in topic.
In example app, when i select file from Google Drive returned value is null.
Box, Dropbox works fine.

**Comment by yoavst** [comment]
You can use this method to check if Google drive selected:

``` java
public static boolean isGoogleDriveDocument(Uri uri) {
return "com.google.android.apps.docs.storage".equals(uri.getAuthority());
}
```

**Comment by jcesarmobile** [comment]
but how do you get the path if isGoogleDriveDocument?

**Comment by yoavst** [comment]
I don't know :/

**Comment by MichalRadwantzor** [comment]
Hi @jcesarmobile, were you able to find a solution for this issue?

**Comment by jcesarmobile** [comment]
Not really a solution, but I get the InputStream from the drive file and store it on a temp folder and use this copy of the drive file instead of the original file

**Comment by MichalRadwantzor** [comment]
Thank u @jcesarmobile

**Comment by danieltigse** [comment]
Can I see the code of this solution?

**Comment by samuelt1** [comment]
i want the code to this solution

**Comment by xiangcman** [comment]
@jcesarmobile  In this case, the name of the file will change

**Comment by jcesarmobile** [comment]
Yeah, it's a copy, so you have to create the file and name it. I don't remember if it's possible to get the original file name and use the same.

**Comment by xiangcman** [comment]
i can get the original file name,can you see your create the file code?

**Comment by jcesarmobile** [comment]
Sorry, I used it in a project long time ago, but don't have the code anymore.

But I probably just googled how to create a file from InputStream

**Comment by xiangcman** [comment]
@jcesarmobile  i have select a mp3 ,the ui three have bean blocked
